Sources/SwifterSwift/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -91,15 +91,12 @@ public extension Optional {
 public extension Optional where Wrapped: Collection {
     /// SwifterSwift: Check if optional is nil or empty collection.
     var isNilOrEmpty: Bool {
-        guard let collection = self else { return true }
-        return collection.isEmpty
+        return (self == nil || self!.isEmpty)
```

Comment:
Why not do `self?.isEmpty ?? true` like below?

Reply:
Yeah we can do that. But initially I thought anyways in second OR condition it would not be nil anyways. So that's why I was force unwrapping. I will do this change. Thanks for the feedback.